lib/Onnxifi/Base.cpp

```diff
@@ -55,15 +55,74 @@ onnxStatus Graph::initGraph(const void *onnxModel, size_t onnxModelSize,
     return ONNXIFI_STATUS_INTERNAL_ERROR;
   }
 
+  onnxNameToInputVar_ = loader->getInputVarsMapping();
+  onnxNameToOutputNode_ = loader->getOutputVarsMapping();
+
+  // Emit IR for the graph and compile it.
+  backendPtr_->getEE().compile(CompilationMode::Infer, function_);
+
   return ONNXIFI_STATUS_SUCCESS;
 }
 
-onnxStatus Graph::run() { return ONNXIFI_STATUS_SUCCESS; }
+onnxStatus Graph::run() {
+  // Copy tensors from the input addresses to the Glow tensors.
+  std::vector<Tensor *> tensors;
```

Comment:
Just curious, when do we use std containers and when do we use llvm containers?

Reply:
llvm one would be preferable, i'll change that to the corresponding vector type.

Reply:
Wait, this one is for external function (RunInference). Maybe std is better.

Reply:
Run inference takes takes llvm types:
```
void ExecutionEngine::run(llvm::ArrayRef<Variable *> vars,
                          llvm::ArrayRef<Tensor *> inputs)
```
which should be fine.